src/server/views/api/queueRetry.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Retry all queue failed jobs
+ * @param {*} req 
+ * @param {*} res 
+ */
+async function handler(req, res) {
+  const { queueName, queueHost } = req.params;
+
+  const {Queues} = req.app.locals;
```

Comment:
```suggestion
  const { Queues } = req.app.locals;
```

For consistency with the destructuring operation above this one.